src/Core/Settings.h

```diff
@@ -812,9 +812,10 @@ class IColumn;
     M(Bool, allow_create_index_without_type, false, "Allow CREATE INDEX query without TYPE. Query will be ignored. Made for SQL compatibility tests.", 0) \
     M(Bool, create_index_ignore_unique, false, "Ignore UNIQUE keyword in CREATE UNIQUE INDEX. Made for SQL compatibility tests.", 0) \
     M(Bool, print_pretty_type_names, false, "Print pretty type names in DESCRIBE query and toTypeName() function", 0) \
+    M(DateTimeOverflowMode, date_time_overflow_mode, DateTimeOverflowMode::IGNORE, "Sets overflow mode for Date, Date32, DateTime, DateTime64 types.", 0) \
```

Comment:
`Sets` is redundant

Please append to the description "Possible values: 'ignore', 'throw', 'saturate'."